modules/gdscript/README.md

```diff
@@ -0,0 +1,92 @@
+# Basic GDScript module architecture
```

Comment:
Inconsistent blank lines in the file. I suggest using 2 blank lines between sections and 1 blank line to separate the body from the header.

```md
## Header

Text.


## Next header

Text.
```